examples/models/export_hf_model.py

```diff
@@ -0,0 +1,178 @@
+    # architecture: stablelm
+    ("stabilityai/stablelm-3b-4e1t", "GOOD"),
+    ("stabilityai/stablelm-2-1_6b", "GOOD"),
+    # architecture: qwen
+    ("Qwen/Qwen-7B-Chat",),
+    ("Qwen/Qwen1.5-7B-Chat",),
+    # architecture: phi2
+    ("microsoft/phi-2", "GOOD"),
+    ("microsoft/phi-1_5", "GOOD"),
+    ("microsoft/phi-1", "GOOD"),
+    # architecture: codeshell
+    ("WisdomShell/CodeShell-7B",),
+    # architecture: internlm2
+    ("internlm/internlm2-7b", "GOOD"),
+    # architecture: minicpm
+    ("openbmb/MiniCPM-2B-sft-bf16",),
+    # architecture: gemma
+    ("google/gemma-2b",),
+    ("google/gemma-7b",),
+    # unknown
+    ("microsoft/biogpt", "GOOD"),
+    ("deepseek-ai/deepseek-llm-7b-chat",),
+    ("01-ai/Yi-6B",),
+    ("BAAI/Aquila-7B", "GOOD"),
+    ("BAAI/Aquila2-7B", "GOOD"),
+    ("THUDM/chatglm3-6b",),
+]
+
+
+def main() -> None:
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "-hfm",
+        "--hf_model_repo",
+        required=False,
+        default=None,
+        help="a valid huggingface model repo name",
+    )
+    parser.add_argument("-o", "--output-dir", default=".", help="output directory")
+    parser.add_argument(
+        "-etx",
+        "--to_et_w_xnn",
+        required=False,
+        action="store_true",
+        help="export ExecuTorch with XNNPACK",
+    )
+    parser.add_argument(
+        "-sg",
+        "--skip_good",
+        required=False,
+        action="store_true",
+        help="skip good models",
+    )
+    args = parser.parse_args()
+
+    if not os.path.exists(args.output_dir):
+        print(f"Error: Output directory '{args.output_dir}' does not exist.")
+        return sys.exit(1)
+
+    hf_model_repos = (
+        HF_LLM_MODLE_REPOS if not args.hf_model_repo else [(args.hf_model_repo,)]
+    )
+
+    device = "cpu"
+    prompt = "What is your favourite condiment?"
+    result_summary = {}
+
+    for repo_config in hf_model_repos:
+        hf_model_repo, status = repo_config[:2] + ("",) * (2 - len(repo_config))
+        if args.skip_good and status == "GOOD":
+            print(f"Skip model '{hf_model_repo}'.")
+            result_summary[hf_model_repo] = "SKIPPED"
+            continue
+
+        model_name = hf_model_repo.split("/")[-1].lower()
+        try:
+            print("\n\n#######################################################")
+            print(f"Load model '{hf_model_repo}' from Hugginface.")
+            print("#######################################################")
+            model = AutoModelForCausalLM.from_pretrained(
+                hf_model_repo, trust_remote_code=True
+            ).to(device)
+            model.eval()
+            tokenizer = AutoTokenizer.from_pretrained(
+                hf_model_repo, trust_remote_code=True
+            )
+            model_inputs = (
+                tokenizer([prompt], return_tensors="pt").to(device)["input_ids"],
+            )
+
+            edge_m = export_to_edge(
```

Comment:
Let's add a torch.no_grad() before this
```suggestion
            with torch.no_grad():
                edge_m = export_to_edge(
```